tests/server.test.ts

```diff
@@ -0,0 +1,46 @@
+import { startServer } from '../src/lib/server';
+const request = require('supertest-as-promised');
+const express = require('express');
+
+// CONSTANTS
+const CONFIGURATION = {
+  port: 3000,
+  https: {
+    certificate: 'tests/assets/test-ssl.cert',
+    privateKey: 'tests/assets/test-ssl.key',
+    port: 3001,
+  },
+};
+
+describe('Initialise things before running application', () => {
+  describe('#startServer', () => {
+    let app;
+    beforeEach(() => {
+      app = express();
+    });
+    test('start http server', async () => {
```

Comment:
Descriptions as a user story (should...) for more consistency throughout our test descriptions